src/message/MessageContainer.js

```diff
@@ -16,29 +19,89 @@ const htmlToDomTree = html => {
   return domTree;
 };
 
-export default class MessageContainer extends React.PureComponent {
+
+class MessageContainer extends React.PureComponent {
```

Comment:
Create a new component or module and extract all Action Sheet related functionality. Maybe `MessageActionSheet'?